### DISCUSSION TITLE
[Meta] Beatmap moderation queue for specific game mode

### DISCUSSION THREAD

**Discussion by stevefan1999-personal** [ppy/osu#28081] [Ideas]
I know this is not something usual and I'm not sure about this being the best place to ask, but I'm interested in making a new game mode based on Sound Voltex in lazer, and I asked someone about whether Konami would get their hands on it. 

I know Konami and Nintendo they are both corporate litigious monsters so sadly speaking, my friend said the worst case scenario is that Konami would intervene immediately with a DMCA if any Konami-copyrighted materials have been used, and probably send a C&D to nuke the game mode itself, rendering all efforts moot (yes, I know I don't have the balls to make it happen yet, as I haven't even started on modding the DSP in the lazer engine). 

In order to try and not to kick the hornets' nest, I think we will need to do it like Beatoraja: displaying a disclaimer citing no Konami-related copyrighted materials to be used (that is, converting IIDX charts into BMS charts and play it inside Beatoraja). This could be easily done in the game mode itself, maybe show it on the default skin with a clear indication of this, but we won't monitor whether or not the end user uphold this rule, so if there are people circulating SDVX converted charts, we don't cockblock them from doing so, because from that point on the game mode itself act as a dumb pipe it's just people who decided to play copyrighted materials over the platform (just so that you don't have to do it necessarily)

But as a further insurance, I think we need to moderate the user generated content from osu's side (not to the end user), i.e. beatmap people submit as well (even when it is in pending state), so that to guarantee that no Konami-related materials (cough cough SDVX maps) exists to eliminate the doubt of copyright violation.

Maybe I will have to split this discussion into multiple smaller discussions but anyway, let me see what we could do first...

**Comment by bdach** [comment]
I have no idea where to begin responding to this. If you wanted to develop a custom ruleset for this yourself, then it is pretty much your problem at this stage if it gets C&D'd. We are not at capacity to add full official support for _any_ new ruleset at this stage.

**Reply by stevefan1999-personal** [reply]
yes yes yes, I know, so I'm focusing more on the user generated content side. Once there is a copyright-infringing beatmap appearing on osu.ppy.sh, regardless of it pending, WIP, ranked or not, at this point Konami can come in and whoop our asses, so there needs to be a way to prevent any Konami maps from even appearing in the listing, i.e. a moderation queue. 

I think this should be done on a case-by-case situation given the game mode/ruleset (whatever it is called), because the sensitivity of the companies involved.

**Reply by bdach** [reply]
https://osu.ppy.sh/wiki/en/Rules/Content_usage_permissions
https://osu.ppy.sh/legal/en/Copyright
